L/libssh/build_tarballs.jl

```diff
@@ -0,0 +1,68 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libssh"
+version = v"0.10.6"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://www.libssh.org/files/$(version.major).$(version.minor)/libssh-$(version).tar.xz", "1861d498f5b6f1741b6abc73e608478491edcf9c9d4b6630eef6e74596de9dc1")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# Necessary for cmake to find openssl on Windows
+if [[ ${target} == *w64* ]]; then
```

Comment:
To be honest `w64` is a bit obscure, I think
```suggestion
if [[ ${target} == x86_64-*-mingw* ]]; then
```
would be slightly clearer: the architecture and the runtime parts of the triple are clearly spelled out

Reply:
Sure, fixed in 52ccbf4c.